src/controller/java/BUILD.gn

```diff
@@ -122,12 +122,47 @@ android_library("java") {
     "zap-generated/chip/devicecontroller/ClusterWriteMapping.java",
   ]
 
+  if (build_java_matter_controller) {
```

Comment:
We should build those android stubs as an component instead of build into CHIPController

Reply:
done， thanks